30-day-roadmap/Birthday List/index.html

```diff
@@ -11,68 +11,31 @@
       rel="stylesheet"
     />
     <link rel="stylesheet" href="styles.css" />
+    <script src="profile.js"></script>
+    <script src="script.js" defer></script>
   </head>
   <body>
     <div class="list-container">
       <h3 class="title">24 birthdays today</h3>
       <ul class="lists">
-        <li class="user-profile">
-          <img
-            class="user-image"
-            src="https://images.unsplash.com/photo-1438761681033-6461ffad8d80?q=80&w=2940&auto=format&fit=crop&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA=="
-            alt="profile-image"
-          />
-          <div class="user-info">
-            <p class="username">Bertie Yates</p>
-            <p class="age">29 years</p>
-          </div>
-        </li>
-        <li class="user-profile">
-          <img
-            class="user-image"
-            src="https://images.unsplash.com/photo-1499996860823-5214fcc65f8f?q=80&w=2866&auto=format&fit=crop&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D"
-            alt="profile-image"
-          />
-          <div class="user-info">
-            <p class="username">Hester Hogan</p>
-            <p class="age">32 years</p>
-          </div>
-        </li>
-        <li class="user-profile">
-          <img
-            class="user-image"
-            src="https://images.unsplash.com/photo-1500648767791-00dcc994a43e?w=800&auto=format&fit=crop&q=60&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1yZWxhdGVkfDl8fHxlbnwwfHx8fHw%3D"
-            alt="profile-image"
-          />
-          <div class="user-info">
-            <p class="username">Larry Little</p>
-            <p class="age">36 years</p>
-          </div>
-        </li>
-        <li class="user-profile">
-          <img
-            class="user-image"
-            src="https://media.istockphoto.com/id/819419874/photo/young-woman-summer-portrait-outdoors.webp?s=170667a&w=0&k=20&c=9SofNH7CyB0SrzLJTwca-o1tl5C07ORHbThgvRk5FHY="
-            alt="profile-image"
-          />
-          <div class="user-info">
-            <p class="username">Sean Waish</p>
-            <p class="age">34 years</p>
-          </div>
-        </li>
-        <li class="user-profile">
-          <img
-            class="user-image"
-            src="https://images.unsplash.com/photo-1489424731084-a5d8b219a5bb?w=800&auto=format&fit=crop&q=60&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1yZWxhdGVkfDF8fHxlbnwwfHx8fHw%3D"
-            alt="profile-image"
-          />
+        <!-- <li class="user-profile">
+          <div class="user-image"></div>
           <div class="user-info">
             <p class="username">Lola Gardner</p>
             <p class="age">29 years</p>
           </div>
-        </li>
+        </li> -->
       </ul>
       <button class="btn">View all</button>
```

Comment:
- [ ] button should view the rest of the profiles as well
- [ ] user should be able to scroll down to view them
- [ ] once view all is clicked the button text should change to "show less"

Reply:
done